okapi-core/src/main/java/org/folio/okapi/service/impl/DockerModuleHandle.java

```diff
@@ -136,18 +136,22 @@ static String setupDockerAddress(StringBuilder socketAddress, String u) {
   private Future<HttpClientResponse> request(HttpMethod method, String url,
                                              MultiMap headers, Buffer body) {
 
-    RequestOptions requestOptions = new RequestOptions()
-        .setMethod(method)
-        .setAbsoluteURI(dockerUrl + url);
-    if (socketAddress != null) {
-      requestOptions.setServer(socketAddress);
-    }
-    return client.request(requestOptions).compose(request -> {
-      if (headers != null) {
-        request.headers().setAll(headers);
+    try {
```

Comment:
Should make the try block smaller to start from line 146?

Reply:
Done.